client/app/services/axios.js

```diff
@@ -14,6 +15,8 @@ axios.interceptors.request.use(config => {
   const apiKey = Auth.getApiKey();
   if (apiKey) {
     config.headers.Authorization = `Key ${apiKey}`;
+  } else {
+    config.headers.common["X-CSRF-TOKEN"] = Cookies.get("csrf_token");
```

Comment:
We should probably:
1. Encapsulate the logic to get the CSRF token, so we can replace it.
2. Not set anything if no `csrf_token` available?

Reply:
1. `flask_wtf` handles the generation and verifications. I'm not sure it can be replaced (nor should it be?)
2. 8babdd0